src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
+* {
+  font-size: 14px;
+  text-decoration: none;
+  box-sizing: border-box;
+}
+@import 'vars';
+
+html{
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
```

Comment:
It's default values - 
```suggestion

```

Reply:
vHere, I was told to set in in html selector by revier

Reply:
Default values, so you can just remove it